include/picongpu/particles/startPosition/OnePositionImpl.hpp

```diff
@@ -42,7 +42,8 @@ namespace picongpu::particles::startPosition::acc
         template<typename T_Particle, typename... T_Args>
         HDINLINE void operator()(T_Particle& particle, T_Args&&...)
         {
-            particle[position_] = T_ParamClass{}.inCellOffset.template shrink<simDim>();
+            constexpr auto initialPosition = T_ParamClass::inCellOffset;
+            particle[position_] = initialPosition.template shrink<simDim>();
```

Comment:
missed that you added the shrink here, in this case maybe add a one-liner to the param that in 2D the last element will be ignored.

Reply:
done!